LeftPanel.qml

```diff
@@ -550,6 +573,15 @@ Rectangle {
             anchors.bottom: parent.bottom
             anchors.bottomMargin: 5
             connected: Wallet.ConnectionStatus_Disconnected
+            onConnectedChanged: {
+                if (networkStatus.connected == Wallet.ConnectionStatus_Disconnected ||
+                    networkStatus.connected == Wallet.ConnectionStatus_Connecting ||
+                    networkStatus.connected == Wallet.ConnectionStatus_WrongVersion) {
+                    //taskbarButton.stop()
+                } else {
+                    //taskbarButton.resume()
+                }
+            }
```

Comment:
@selsta 
Could you please check how to implement this in C++?
https://doc.qt.io/qt-5/qwintaskbarprogress.html#stop
https://doc.qt.io/qt-5/qwintaskbarprogress.html#resume

There is no function available in QML (https://doc.qt.io/qt-5/qml-qtwinextras-taskbarbutton.html#progress.stopped-prop)

Reply:
What happens when you stop the taskbarButton? What's the difference to setting visible to false?

Reply:
When you stop, the progress bar in taskbar button turns red, indicating an error (in this case, a disconnection, but we could also add other errors in future, like not enough space in disk).

![image](https://user-images.githubusercontent.com/45968869/145319063-e575425f-c27a-459d-aa5d-14a1d4134ca3.png)

When visible is false, the progress bar doesn't appear on taskbar button.

Reply:
See e.g. https://code.woboq.org/qt5/qtwinextras/examples/winextras/quickplayer/qml/main.qml.html#91

Reply:
taskbarButton.stopped and taskbarButton.paused don't work:
` qrc:/LeftPanel.qml:580: Error: Cannot assign to non-existent property "stopped"`
See: https://www.qtcentre.org/threads/62358-QtWinExtras-TaskbarButton-Progress-not-turning-red

Reply:
Did you try `taskbarButton.progress.stop()`?

Reply:
You're right, it works. I was using taskbarButton.stop().